File: python/7 kyu/91.py
# https://www.codewars.com/kata/57efcb78e77282f4790003d8/train/python

# Passed

import math

def how_many_times(annual_price, individual_price):
    return math.ceil(annual_price / individual_price)

output = how_many_times(40,15)
print(output)

File: nodejs/7 kyu/91.js
// https://www.codewars.com/kata/59a2a3ba5eb5d4e609000055/train/javascript

// Passed

function findArray(arr1, arr2) {
  const result = arr2
    .map((idx) => (arr1.length > idx ? arr1[idx] : null))
    .filter((elem) => elem !== null);
  return result;
}

const output = findArray([0, 1, 5, 2, 1, 8, 9, 1, 5], [1, 4, 7]);
console.log(output);